compiler/rustc_codegen_ssa/src/sir.rs

```diff
@@ -341,102 +289,292 @@ impl SirFuncCx<'tcx> {
     }
 
     /// Converts a MIR statement to SIR, appending the result to `bb`.
-    pub fn lower_statement(&mut self, bb: ykpack::BasicBlockIndex, stmt: &mir::Statement<'_>) {
+    pub fn lower_statement<Bx: BuilderMethods<'a, 'tcx>>(
+        &mut self,
+        bx: &Bx,
+        bb: ykpack::BasicBlockIndex,
+        stmt: &mir::Statement<'tcx>,
+    ) {
+        self.push_stmt(
+            bb,
+            ykpack::Statement::Debug(with_no_trimmed_paths(|| format!("{:?}", stmt))),
```

Comment:
Silly question perhaps, but given that `format!` is returning a `String`, this `Debug` information might end up being quite expensive?

Reply:
I was actually meant to strip this debugging stuff out until I can think of a less expensive way to do it. Oops.

I'll do that now.

Reply:
Fixed in 98f57e97b9102848faf8d929cd7e9fc543bc7bc2, but note that there was also a change in the yk repo.